private-prediction/a - Secure Model Serving.ipynb

```diff
@@ -4,20 +4,20 @@
    "cell_type": "markdown",
```

Comment:
<p>Part of this PR, can we include in the Secure FL Notebook (Part 1 with the first secure fl example, not the exercise) the last step where we are saving the secure FL model (<code>tf.keras.models.save_model(secure_fl_model)</code>). The model would be saved in the privave_prediction folder and <code>trained_model</code> would point to it. </p><p> </p><p>This way in the PR, we have the end to end process running( FL -&gt; private prediction). If needed @<a href="https://github.com/jvmancuso" target="_blank">jvmancuso</a> and I will update (e.g. new dataset, new model etc).</p><p> </p><p>Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dropoutlabs/tf-world-tutorial/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I think we should use the saving code that I added as part of tf-encrypted. That code doesn't exist in tf world yet. I believe @jvmancuso will port the code over when he's done making modifications to the FL example.

Reply:
Ok great, let's do that then